cypress/integration/App.spec.ts

```diff
@@ -14,6 +14,8 @@ describe('Open aukiolot app', () => {
   it('Has no detectable a11y violations on resource page load', () => {
     cy.injectAxe();
     cy.get('[data-test=resource-info]', { timeout: 5000 }).should('be.visible');
+
+    cy.wait(1000); // Wait for HDS spinner cleanup.
```

Comment:
I wonder if there's a better way to do this wait? Could we check that the spinner is no longer there? Or is there are problem in that because it might not have appeared yet, so we would get false positives? I wonder if there's a method in cypress for checking this with some initial waiting time.

Reply:
The clean-up callback is called one second after spinner disappears